plugins/harness-chaos/src/components/Router.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { Routes, Route } from 'react-router';
+import { Entity } from '@backstage/catalog-model';
+import {
+  MissingAnnotationEmptyState,
+  useEntity,
+} from '@backstage/plugin-catalog-react';
+import ChaosExperiments from './ChaosExperiments';
+import { isV1Compatible, isV2Compatible } from '../utils/getCompatibleVersion';
+
+/** @public */
+export const isHarnessChaosAvailable = (entity: Entity | undefined) => {
+  if (!entity) return true;
```

Comment:
```suggestion
  if (!entity) return false;
```

Reply:
Thank you for this, had flipped it for testing something, forgot to revert. Fixed it now.